code/modules/projectiles/gun_attachables.dm

```diff
@@ -1663,6 +1665,42 @@ inaccurate. Don't worry if force is ever negative, it won't runtime.
 
 	return FALSE
 
+/obj/item/attachable/lace
+	name = "pistol lace"
+	desc = "A simple lace to wrap around your wrist."
+	icon_state = "lace"
+	attach_icon = "lace_a"
+	slot = "muzzle" //so you cannot have this and RC at once aka balance
+	flags_attach_features = ATTACH_REMOVABLE|ATTACH_ACTIVATION
+	attachment_action_type = /datum/action/item_action/toggle
+
+/obj/item/attachable/lace/activate_attachment(mob/living/user, turn_off)
+	if(lace_deployed)
+		lace_deployed = FALSE
+		DISABLE_BITFIELD(master_gun.flags_item, NODROP)
+		to_chat(user, "<span class='notice'>You feel the [src] loosen around your wrist!</span>")
+		playsound(user, 'sound/weapons/fistunclamp.ogg', 25, 1, 7)
+		icon_state = "lace"
+		attach_icon = "lace_a"
+	else
+		if(user.action_busy)
+			return
+		if(!do_after(user, 0.5 SECONDS, TRUE, src, BUSY_ICON_BAR))
+			return
+		if(lace_deployed)
+			return
+		lace_deployed = TRUE
+		to_chat(user, "<span class='notice'>You deploy [src].</span>")
```

Comment:
```suggestion
		to_chat(user, "<span class='notice'>You deploy the [src].</span>")
```

Reply:
THANKS a bunch slayer truly!! (((my bad is english mistake often hehe)))